android/src/com/mapswithme/util/Constants.java

```diff
@@ -23,8 +23,8 @@
     public static final String FB_OM_COMMUNITY_HTTP = "https://www.facebook.com/OrganicMaps";
     public static final String FB_OM_COMMUNITY_NATIVE = "fb://profile/102378968471811";
     public static final String TWITTER = "https://twitter.com/OrganicMapsApp";
-    public static final String TELEGRAM = "https://t.me/OrganicMaps";
-    public static final String INSTAGRAM = "https://www.instagram.com/organicmaps.app";
+    public static final String TELEGRAM = String TG_LINK;
```

Comment:
How does it work?

Reply:
I don't know. I've just tried to make similar with other codes. So I'm not sure this works correctly.